### PR TITLE
feat(NRQL): added guideposts to new NRQL docs (71566) 

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

**Comment by jeff-colucci** [comment]
@nbaenam The links for the tutorial at the end of each guidepost won't work until we publish the tutorial. My plan is to have the tutorial merged in a morning PR and then merge this one in an afternoon PR.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx [hunk 1]
```diff
@@ -56,4 +56,6 @@ While charts are great for reporting on individual pieces, dashboards are the be
...
 * [Duplicate](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-clone) the dashboard to use as a foundation for an entirely new one
-* [Edit](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#add-pages-edit) your dashboard components to add new data streams, modify existing ones, or remove older ones that are no longer relevant
\ No newline at end of file
+* [Edit](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#add-pages-edit) your dashboard components to add new data streams, modify existing ones, or remove older ones that are no longer relevant
+
+Have you seen our [intro to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) doc and learned [how to use NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/)? If you have, you're ready to start our [guided NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/)!
```
Comment (nbaenam) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:61]: I wouldn't use the word 'doc', I think that I'd remove it or write "document"

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx [hunk 2]
```diff
@@ -296,4 +296,6 @@ Here are some more query examples:
 
 <Callout variant="important">
   To explore your data without having to use NRQL, use the [metrics and events data explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/). Learn more about [querying data in New Relic](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/).
-</Callout>
\ No newline at end of file
+</Callout> 
+
+Ready to learn more? Be sure to check out our [intro to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) doc if you haven't already, or learn [how to use charts and dashboards with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql/). If you want to start using NRQL right away, jump straight into our [guided NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
```
Comment (nbaenam) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx:301]: Same as before with the 'doc' word.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx [hunk 3]
```diff
@@ -225,3 +225,5 @@ NRQL lets you query nearly every type of our telemetry data, including:
 <Callout variant="tip">
   Some data, like relationships between monitored entities, is not available via NRQL but is available using our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph).
 </Callout>
+
+Ready to learn more? Check out our docs on [how to use NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and [how to use charts and dashboards with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql/). If you want to start using NRQL instead, jump straight into our [guided NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
```
Comment (nbaenam) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx:229]: Same with 'our docs'. I think I'd remove 'our docs', I think it's better 'Check out how to use....'

**Review by nbaenam — APPROVED** [review]

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx (hunk 1 above)
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx:61]: Good call, will remove!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx (hunk 2 above)
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx:301]: Thank you!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx (hunk 3 above)
Comment (jeff-colucci) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx:229]: Good suggestion, I will make a change. Thank you!

**Review by nbaenam — APPROVED** [review]

**Pull request merged**